Services/DataX.Config/DataX.Config.Input.EventHub/Processor/CreateEventHubConsumerGroup.cs

```diff
@@ -77,15 +77,6 @@ public override async Task<FlowGuiConfig> HandleSensitiveData(FlowGuiConfig guiC
                 guiConfig.Input.Properties.InputResourceGroup = secretId;
             }
 
-            // Replace Info Databricks Token
-            var infoDatabricksToken = guiConfig?.DatabricksToken;
-            if(!string.IsNullOrEmpty(infoDatabricksToken) && !KeyVaultUri.IsSecretUri(infoDatabricksToken))
-            {
-                var secretName = $"{guiConfig.Name}-info-databricksToken";
-                var secretId = await KeyVaultClient.SaveSecretAsync(runtimeKeyVaultName, secretName, infoDatabricksToken, Configuration[Constants.ConfigSettingName_SparkType]);
```

Comment:
where did this go?

Reply:
it's removed by accident. will get this back.